tests/DCPS/XTypes/Publisher.cpp

```diff
@@ -92,6 +92,23 @@ void write_additional_prefix_field_struct(const DataWriter_var& dw)
   }
 }
 
+void write_base_appendable_struct(const DataWriter_var& dw)
+{
+  BaseAppendableStructDataWriter_var typed_dw = BaseAppendableStructDataWriter::_narrow(dw);
+
+  BaseAppendableStruct apfs;
```

Comment:
Looks like the variable name `apfs` is copied over from `write_additional_prefix_field_struct` function. Can you change to use a better name, something like `bas` for example?